File: src/Movie.java
import java.util.ArrayList;

public class Movie {
    private String title;
    public ArrayList<MovieScreening> screenings;

    public Movie(String title) {
        this.screenings = new ArrayList<>();
        this.title = title;
    }

    public void addScreening(MovieScreening screening){
        this.screenings.add(screening);
    }

    @Override
    public String toString() {
        return "Movie [title=" + title + "]";
    }
}

File: Domain/Movie.cs
namespace Domain
{
    public class Movie
    {
        public string Title { get; set; }
        public List<MovieScreening> Screenings { get; set; }

        public Movie(string title)
        {
            Title = title;
            Screenings = new List<MovieScreening>();
        }

        public void AddScreening(MovieScreening screening)
        {
            Screenings.Add(screening);
        }
    }
}